README.md

```diff
@@ -118,6 +118,21 @@ openssl::export::pem_key { 'foo':
 }
 ```
 
+### Dynamic refresh of exported files
+
+If you want Puppet to refresh the PKCS#12, PEM/x509 or PEM key file in case the input files changed, set the dynamic mode on and handle over desired resources for subscription:
```

Comment:
```suggestion
If you want Puppet to refresh the PKCS#12, PEM/x509 or PEM key file in case the input files changed, set the dynamic mode on and list desired resources for subscription:
```